server/api/profile.js

```diff
@@ -1,12 +1,25 @@
 const express = require("express");
 const { createProfile } = require("../services/database/profile");
 const router = express.Router();
+const profileDb = require("../services/database/profile");
 
 /** route is /profile/ (status is defined as prefix in index.js) */
 router.post("/", (req, res) => {
-  const db_newProfile = req.body;
-  createProfile(db_newProfile);
-  res.status(201).send("Profile created");
+	const db_newProfile = req.body;
+	createProfile(db_newProfile);
+	res.status(201).send("Profile created");
+});
+
+router.get("/", (req, res) => {
```

Comment:
This api endpoint should be reached at /profiles to follow naming convention for REST APIs.